imports/decorators/infiniteScroll.js

```diff
@@ -30,7 +29,7 @@ export default (reducer = defaultReducer, options = defaultOptions) => (WrappedC
       const scrollPosition = window.scrollY;
       const deviceHeight = window.outerHeight;
       const contentHeight = document.body.clientHeight;
-      const threshold = options.percent * 0.01;
+      const threshold = (options.percent || 70) * 0.01;
```

Comment:
I added this part so that there is still a default present if you specify options.  Would this be the best way to do it, as defaultOptions is overwritten by specifying options on the decorator?